src/computability/epsilon_NFA.lean

```diff
@@ -0,0 +1,131 @@
+/-
+Copyright (c) 2021 Fox Thomson. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Fox Thomson
+-/
+
+import computability.NFA
+
+/-!
+# Epsilon Nondeterministic Finite Automata
+This file contains the definition of an epsilon Nondeterministic Finite Automaton (`ε_NFA`), a state
+machine which determines whether a string (implemented as a list over an arbitrary alphabet) is in a
+regular set by evaluating the string over every possible path, also having access to ε-transitons,
+which can be followed without reading a character.
+Since this definition allows for Automata with infinite states, a `fintype` instance must be
```

Comment:
```suggestion
Since this definition allows for automata with infinite states, a `fintype` instance must be
```
At least in the Handbook of Formal Languages, the words automaton and automata are always written in lowercase (except for titles).